### PR TITLE
AC-744 Refactored the Utility Package to kotlin

### PR DESCRIPTION
**Pull request by gauravdewat**
AC-744 Refactoring utilities from java to kotlin

## Description of what I changed

All the util files have been refactored to kotlin. ConceptEntity class is also refactored to kotlin. 

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-744

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by LuGO0** [comment]
@gauravdewat Its generally a bad practice to close pull requests just because of some issues but since the Old PR didnt have some useful discussions its okay for now !!.
But I would suggest you not to do this again :)

File: openmrs-client/src/main/java/org/openmrs/mobile/utilities/ImageUtils.kt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            return inSampleSize
+        }
+
+    @JvmStatic
+        fun resizePhoto(photo: Bitmap): Bitmap {
+            val HEIGHT = 500.0
+            val WIDTH = 500.0
+            val height = photo.height
+            val width = photo.width
+            val aspectRatio = Math.min(HEIGHT / height, WIDTH / width)
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/utilities/ImageUtils.kt:52]: Use `coerceAtMost` instead.

File: openmrs-client/src/main/java/org/openmrs/mobile/utilities/InputField.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
+package org.openmrs.mobile.utilities
+
+import android.os.Parcel
+import android.os.Parcelable
+import java.io.Serializable
+
+
+class InputField(var concept: String) : Serializable, Parcelable {
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/utilities/InputField.kt:8]: The class variable `concept` may be null as suggested at line 29
```suggestion
class InputField(var concept: String?) : Serializable, Parcelable {
```

**Comment by rishabh-997** [comment]
What are tge updates on this PR @gauravdewat

**Comment by f4ww4z** [comment]
Closing due to inactivity. @gauravdewat Ping me if you want to resume working on it.

**Pull request closed without merging**